src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -8448,10 +8448,10 @@ private ExpressionSyntax ParseSubExpressionCore(uint precedence)
             {
                 var questionToken = this.EatToken();
 
-                var colonLeft = this.ParseSubExpression(nullCoalescingPrecedence - 1);
+                var colonLeft = this.ParseExpressionCore();
```

Comment:
Is ```nullCoalescingPrecedence - 1``` equal to 0? If not, what besides query expressions can be affected by this change? There are no new tests added for conditional expression. Would it make sense to make this change only for strict mode?